docs/software/webserver_stack.md

```diff
@@ -520,7 +523,8 @@ It is designed to be secure, fast, standards-compliant, and flexible while being
 
 === "Official documentation"
 
-    - <https://www.lighttpd.net>
+    - Documentation: [Welcome to Lighttpd](https://redmine.lighttpd.net/projects/lighttpd/wiki)
+    - Website : <https://www.lighttpd.net>
```

Comment:
We have a few different ways of adding official external links now. Let's agree with a consistent way. E.g.: https://dietpi.com/docs/software/cloud/
- `See also https://www.tonido.com/` below the tabs section as simple text line.
- https://dietpi.com/docs/software/cloud/#official-links: A set of links below the tabs section with an own `###` header.
- And sometimes they are inside a tab.

I'll open a new issue/tasks to discuss how to align those.

Reply:
ok. agree, let's discuss